# jaguar arm motor command message from DrRobot Robot.
uint32 motorID		# motor ID number
float32 jointAngle	        # joint target angle


